masschallenge/impact-api
# [6986] expose id on location and expert graphQL data

**Changes introduced in [AC 6986]( https://masschallenge.atlassian.net/browse/AC-6986):**
- Mentor office hours create/edit modal.

**How to test:**
- Checkout to this branch
- With Prod db masquerade as a mentor 
- Go to the [office hour page](http://localhost:8181/newofficehour)
- Create an Office hour
- Ensure all done of ticket checks out

linked pr
[AC 6986 Accelerate]( https://github.com/masschallenge/accelerate/pull/2632)
